crates/eframe/src/native/epi_integration.rs

```diff
@@ -75,7 +77,9 @@ pub fn window_builder(
         .with_maximized(*maximized)
         .with_resizable(*resizable)
         .with_transparent(*transparent)
-        .with_window_icon(window_icon);
+        .with_window_icon(window_icon)
+        // We must not show the window until AccessKit is initialized.
+        .with_visible(false);
```

Comment:
Already done in https://github.com/emilk/egui/pull/2279, but it would be good with this comment with the added motivation